pkg/monitoring/source_envelope.go

```diff
@@ -133,6 +126,24 @@ func (s *envelopeSource) Fetch(ctx context.Context) (interface{}, error) {
 	return envelope, nil
 }
 
+func (s *envelopeSource) getLinkBalance(ctx context.Context, tokenVault solana.PublicKey) (*big.Int, error) {
+	linkBalanceRes, err := s.client.GetTokenAccountBalance(ctx, tokenVault, rpc.CommitmentConfirmed)
+	if err != nil {
+		return nil, fmt.Errorf("failed to read the feed's link balance: %w", err)
+	}
+	if linkBalanceRes == nil || linkBalanceRes.Value == nil {
+		return nil, fmt.Errorf("link balance not found for token vault")
+	}
+	linkBalance, success := big.NewInt(0).SetString(linkBalanceRes.Value.Amount, 10)
+	if !success {
+		return nil, fmt.Errorf("failed to parse link balance value: %s", linkBalanceRes.Value.Amount)
+	}
+	if linkBalance.Cmp(big.NewInt(0)) == 0 {
```

Comment:
Would it work to move `big.NewInt(0)` into a constant/global variable to prevent allocating a new big int every time?